rando_modules/logic.py

```diff
@@ -540,10 +547,9 @@ def _get_limit_items_to_dungeons(
                 False,
                 False
             )
-        add_to_inventory([
-            "CrystalBerry",
-            "WaterStone"
-        ])
+        if area_name in additional_starting_items:
+            for item in additional_starting_items[area_name]:
```

Comment:
```suggestion
            add_to_inventory(additional_starting_items[area_name])
```  
and remove line below.
Reasoning: add_to_inventory has handling for an argument of type list